saturday-afternoon/components/flights/Airlines.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { Fragment } from "react";
+import { Col, Row, Avatar } from "antd";
+import { createFragmentContainer, graphql } from "react-relay";
+import idx from "idx";
+
+import type { Airlines as AirlinesType } from "./__generated__/Airlines.graphql";
+
+type Props = {
+  data: AirlinesType
+};
+
+const Airlines = ({ data: { legs } }: Props) => (
+  <Fragment>
+    {legs &&
+      legs.map(leg => {
+        if (leg) {
```

Comment:
Can we save one level of indentation with this coding style?
```js
if (!leg) return null;
// and now the happy path
```

Reply:
yes good idea